Healthy/Classes/Extensions/UICollectionView+Helpers.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+extension UICollectionView {
+    func register<T: UICollectionViewCell>(_: T.Type) {
+        let bundle = Bundle(for: T.self)
```

Comment:
```suggestion
    func register<T: UICollectionViewCell>(_: T.Type, bundle: Bundle? = nil) {
        let bundle = bundle ?? Bundle(for: T.self)
```
When we migrate to SPM, with will break. Please cosider adding nullable Bundle.